augur/tasks/github/util/github_task_session.py

```diff
@@ -4,6 +4,27 @@
 from augur.application.db.session import DatabaseSession
 
 
+class GithubTaskManifest:
```

Comment:
This file should probably be renamed in the future considering we renamed the corresponding class. It's not worth holding up the pr to rename it though.

Reply:
My plan was to rename it after we fully factor out the `GithubTaskSession` logic. Currently this file still contains the `GithubTaskSesssion` class so we can slowly convert over to using the `GithubTaskManifest` instead